src/Slices/ServiceInstanceDetails/Test/InstanceActions.test.tsx

```diff
@@ -0,0 +1,480 @@
+
+      await userEvent.selectOptions(operationsSelect, options[1]);
+
+      expect(operationsSelect).toHaveValue("clear candidate");
+
+      await userEvent.click(confirmButton);
+    });
+
+    expect(screen.queryByRole("dialog")).toBeNull();
+    expect(
+      screen.queryByTestId("error-toast-expert-state-error-message"),
+    ).toBeNull();
+  });
+
+  it("Expert actions - Destroy", async () => {
+    const component = setupServiceInstanceDetails(true);
+
+    render(component);
+
+    expect(
+      await screen.findByRole("region", { name: "Instance-Details-Loading" }),
+    ).toBeInTheDocument();
+
+    expect(
+      await screen.findByRole("region", { name: "Instance-Details-Success" }),
+    ).toBeInTheDocument();
+
+    // expect to find the expert actions dropdown
+    const expertDropdown = screen.getByRole("button", {
+      name: "Expert-Actions-Toggle",
+    });
+
+    await act(async () => {
+      await userEvent.click(expertDropdown);
+    });
+
+    await act(async () => {
+      const destroyAction = screen.getByRole("menuitem", {
+        name: /destroy/i,
+      });
+
+      await userEvent.click(destroyAction);
+    });
+
+    await act(async () => {
+      const confirmButton = screen.getByRole("button", {
+        name: /yes/i,
+      });
+
+      await userEvent.click(confirmButton);
+    });
+
+    expect(
+      screen.queryByTestId("error-toast-expert-state-error-message"),
+    ).toBeNull();
+    await waitFor(() =>
+      expect(mockedUsedNavigate).toHaveBeenCalledWith(
+        "/console/lsm/catalog/mobileCore/inventory?env=aaa",
+      ),
+    );
+  });
+
+  it("Normal Instance Actions Enabled - delete action", async () => {
+    const component = setupServiceInstanceDetails();
+
+    render(component);
+
+    expect(
+      await screen.findByRole("region", { name: "Instance-Details-Loading" }),
+    ).toBeInTheDocument();
+
+    expect(
+      await screen.findByRole("region", { name: "Instance-Details-Success" }),
+    ).toBeInTheDocument();
+
+    expect(
+      screen.queryByRole("button", { name: /expert actions/i }),
+    ).not.toBeInTheDocument();
+
+    await act(async () => {
+      // expect to find action dropdown
+      const actionDropdown = screen.getByRole("button", {
+        name: "Actions-Toggle",
+      });
+
+      await userEvent.click(actionDropdown);
+    });
+
+    const actions = screen.getAllByRole("menuitem");
+
+    expect(actions).toHaveLength(5);
+
+    actions.forEach((action) => {
+      expect(action).toBeEnabled();
+    });
+
+    // delete instance
+    await act(async () => {
+      await userEvent.click(actions[3]);
+    });
+
+    expect(
+      screen.getByText(
+        /are you sure you want to delete instance core1 of service entity mobilecore\?/i,
+      ),
+    ).toBeVisible();
+
+    await act(async () => {
+      const confirmButton = screen.getByRole("button", {
+        name: /yes/i,
+      });
+
+      await userEvent.click(confirmButton);
+    });
+
```

Comment:
There could be assertion that the instance got deleted, currently we only know that dialog got closed and there is no errors